sdk/java-sdk/src/main/java/com/akkaserverless/javasdk/logging/LogbackJsonLayout.java

```diff
@@ -35,5 +38,23 @@ public LogbackJsonLayout() {
   public void addCustomDataToJsonMap(
       java.util.Map<String, Object> map, ch.qos.logback.classic.spi.ILoggingEvent event) {
     add("severity", true, String.valueOf(event.getLevel()), map);
+    if (event.getMDCPropertyMap().containsKey(ErrorHandling.CorrelationIdMdcKey())) {
+      // automatically include correlation id in message if present (for now)
+      String correlationID = event.getMDCPropertyMap().get(ErrorHandling.CorrelationIdMdcKey());
+      if (this.includeMessage) {
+        add(
+            MESSAGE_ATTR_NAME,
+            this.includeMessage,
+            map.get(MESSAGE_ATTR_NAME) + " [" + correlationID + "]",
+            map);
+      }
+      if (this.includeFormattedMessage) {
+        add(
+            FORMATTED_MESSAGE_ATTR_NAME,
+            this.includeFormattedMessage,
+            map.get(FORMATTED_MESSAGE_ATTR_NAME) + " [" + correlationID + "]",
+            map);
```

Comment:
wonder if we are going over board and making this more magic than necessary?

We always log these with `ErrorHandling.withCorrelationId { correlationId =>` right? Can't we just include the correlationId in the log message explicitly. Maybe even skip the idea of including it in MDC altogether?

Reply:
Yeah, definitely questionable, I think MDC is the nice thing, so I'd rather cut it out of the log entries then, the only place that is not visible is the console ui (as it is right now).

Reply:
Reverted that now, so only in MDC